src/components/my-map/my-map.js

```diff
@@ -1,26 +1,29 @@
 import './my-map.scss';
+import './configuration/evt-widget-config-marker-color/evt-widget-config-marker-color';
```

Comment:
Unnecessary dependency/link. Components are isolated and we should not suggest this as a good pattern (as usually any import without bindings). This component should be declared and imported together with the others.